microprofile/WFLY-12823_EAP7-1375_quickstart-for-mp-rest-client-1.3.adoc

```diff
@@ -0,0 +1,71 @@
+= WFLY-12823 / EAP7-1375 Quickstart for MP REST-client 1.3
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile REST client 1.3 use in
+WildFly / JBoss EAP. The example will cover all basic functionalities of MicroProfile
+REST Client 1.3, namely: definitions of REST client interfaces, definitions of custom
+exception mappers, CDI injection of the client interfaces, and programmatic lookup of the
+client interfaces.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1375[EAP7-1375]
+* https://issues.jboss.org/browse/WFLY-12823[WFLY-12823]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1368
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* Quickstarts
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
```

Comment:
Please describe the steps of quickstart which will user go through. These key points are IMHO not sufficient for analysis document.